### DISCUSSION TITLE
Hiding previous releases?

### DISCUSSION THREAD

**Discussion by flo-abgrund** [knowledgefutures/pubpub#2759] [Support] [answered]
Hi there,
This issue probably has been raised several times already but I couldn't find a definitive answer to it. Is there a way to hide previous releases of a pub from the public? Or, if that is not possible, can I clone the current release of a pub and delete the original one with the different releases in its history?
I am using pubpub as a blog accompanying a research project and prematurely published a raw draft. 

Thank you all already for helping out!

**Comment by gabestein** [comment, marked as answer]
Hello — it's not currently possible to hide prior releases. In emergencies, we can delete them in the backend, but for the most part we prefer not to, to preserve the record. In this case, if you haven't shared the link yet, you can probably do just as well copying and pasting the contents of a draft into a new pub.
